docs/faq.rst

```diff
@@ -353,6 +353,10 @@ How can I restore huge file(s) over a unstable connection?
 If you can not manage to extract the whole big file in one go, you can extract
 all the part files (see above) and manually concatenate them together.
 
+How can I backup the root partition (/)?
+----------------------------------------
+Backing up your entire root partition works just fine, but remember to exclude directories that makes no sense to backup, such as /dev, /proc, /sys and /run.
```

Comment:
Yeah, the phrasing is way off, was a bit tired. I'll rewrite it and add /tmp to the list.